charts/photoprism/Chart.yaml

```
@@ -0,0 +1,22 @@
+apiVersion: v2
+appVersion: 20201122
```

```suggestion
appVersion: "20201122"
```

This needs quotes, or else the `ct` validation will fail 🙁